lib/matplotlib/tests/test_streamplot.py

```diff
@@ -92,11 +92,11 @@ def test_direction():
 
 def test_streamplot_limits():
     ax = plt.axes()
-    x = np.linspace(-5, 10, 20)
-    y = np.linspace(-2, 4, 10)
+    x = np.linspace(-5, 10, 20) + 25
+    y = np.linspace(-2, 4, 10) + 32
     y, x = np.meshgrid(y, x)
-    trans = mtransforms.Affine2D().translate(25, 32) + ax.transData
-    plt.barbs(x, y, np.sin(x), np.cos(y), transform=trans)
+#    trans = mtransforms.Affine2D().translate(25, 32) + ax.transData
```

Comment:
just delete this?

Reply:
Oops, meant to re-Instate this actually.  Thanks....